pkgs/development/libraries/libusbgx/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, fetchFromGitHub, cmake, bash-completion, pkg-config, libconfig, autoreconfHook }:
+stdenv.mkDerivation {
```

Comment:
```suggestion
{ stdenv, lib, fetchFromGitHub, cmake, bash-completion, pkg-config, libconfig, autoreconfHook }:

stdenv.mkDerivation {
```

Reply:
I'm not willing to make changes not documented in https://nixos.org/manual/nixpkgs/stable/#chap-conventions or implemented by an autoformatting tool.